taskfiles/charts.yml

```diff
@@ -0,0 +1,129 @@
+version: '3'
+
+env:
```

Comment:
This block can be removed. `GOFLAGS` doesn't apply any more and `GOPRIVATE` is set by `flake.nix`

Reply:
Ok